### ISSUE TITLE
histogram plot from plot.model_performance() is in reverse order

### ISSUE THREAD

**Issue by llendway** [ModelOriented/DALEX#400]
The code below uses the example from the plot.model_performance() code. When used with multiple models and geom="histogram", the labels are incorrect (in reverse order, it seems).

library(patchwork)
library(DALEX)

titanic_ranger_model2 <- ranger(survived~gender + fare, data = titanic_imputed,
                                num.trees = 50, probability = TRUE)
explainer_ranger2  <- explain(titanic_ranger_model2, data = titanic_imputed[,-8],
                              y = titanic_imputed$survived,
                              label = "ranger2")
mp_ranger2 <- model_performance(explainer_ranger2)

titanic_glm_model <- glm(survived~., data = titanic_imputed, family = "binomial")
explainer_glm <- explain(titanic_glm_model, data = titanic_imputed[,-8],
                         y = titanic_imputed$survived, label = "glm",
                    predict_function = function(m,x) predict.glm(m,x,type = "response"))
mp_glm <- model_performance(explainer_glm)

titanic_lm_model <- lm(survived~., data = titanic_imputed)
explainer_lm <- explain(titanic_lm_model, data = titanic_imputed[,-8],
                        y = titanic_imputed$survived, label = "lm")
mp_lm <- model_performance(explainer_lm)

# The histogram labels are reversed - the bottom graph should be ranger2 and the top glm.
plot(mp_glm, mp_ranger2, mp_lm, geom = "histogram") +
  plot(mp_glm, mp_ranger2, mp_lm, geom = "boxplot")

# This is clear after looking at the graph of only ranger2.
plot(mp_ranger2, geom = "histogram") +
  plot(mp_ranger2, geom = "boxplot")

**Comment by hbaniecki** [comment]
Indeed, this is due to this line:

https://github.com/ModelOriented/DALEX/blob/922519cd656664a539270ae54e61998dc0b9b67d/R/plot_model_performance.R#L141-L142

@pbiecek any idea why it is there? I found a mismatch of colors in EMA too. 
![image](https://user-images.githubusercontent.com/32574004/112607237-46516c80-8e19-11eb-943b-d1b66e055083.png)
![image](https://user-images.githubusercontent.com/32574004/112607247-494c5d00-8e19-11eb-96d5-22bb1295045b.png)

Can we delete it?

**Comment by pbiecek** [comment]
thanks,
in fact labels were not consistent
and it was a problem not only in model_performance() but also model_parts()

it should be fixed in  24ba9cf

```
library(patchwork)
library(DALEX)
library(ranger)

titanic_ranger_model2 <- ranger(survived~gender + fare, data = titanic_imputed,
                                num.trees = 50, probability = TRUE)
explainer_ranger2 <- explain(titanic_ranger_model2, data = titanic_imputed[,-8],
                             y = titanic_imputed$survived,
                             label = "ranger2")
mp_ranger2 <- model_performance(explainer_ranger2)

titanic_glm_model <- glm(survived~., data = titanic_imputed, family = "binomial")
explainer_glm <- explain(titanic_glm_model, data = titanic_imputed[,-8],
                         y = titanic_imputed$survived, label = "glm",
                         predict_function = function(m,x) predict.glm(m,x,type = "response"))
mp_glm <- model_performance(explainer_glm)

titanic_lm_model <- lm(survived~., data = titanic_imputed)
explainer_lm <- explain(titanic_lm_model, data = titanic_imputed[,-8],
                        y = titanic_imputed$survived, label = "lm")
mp_lm <- model_performance(explainer_lm)
mp_pa <- model_parts(explainer_lm)
mp_pa2 <- model_parts(explainer_glm)
mp_pa3 <- model_parts(explainer_ranger2)

plot(mp_glm, mp_ranger2, mp_lm, geom = "histogram") +
  plot(mp_glm, mp_ranger2, mp_lm, geom = "boxplot") +
  plot(mp_pa2, mp_pa3, mp_pa)
```

**Comment by pbiecek** [comment]
fixed in https://github.com/ModelOriented/DALEX/commit/24ba9cfeedd8596917d6fac1102e2cafa4aca429
